main_project/our_module.rb

```diff
@@ -306,10 +306,7 @@ def create_project_with_name
     create_project_commit
   end
 
-  def open_project(project_name)
-    @wait.until {@driver.find_element(:css => '#projects-index').displayed?}
-    projects = @driver.find_elements(:css => '.project.root.leaf')
-
+  def is_project_in_list?(project_name, projects)
```

Comment:
I would suggest you to change it

1. get all projects titles
2. verify that this array includes project name you're looking for